doc/source/notebooks/basics/regression.ipynb

```diff
@@ -557,16 +357,18 @@
     "                 mean[:,0] + 1.96 * np.sqrt(var[:,0]),\n",
     "                 color='C0', alpha=0.2)\n",
     "\n",
-    "plt.plot(xx, samples[:, :, 0].T, 'C0', linewidth=.5)\n",
+    "plt.plot(xx, samples[:, :, 0].numpy().T, 'C0', linewidth=.5)\n",
     "plt.xlim(-0.1, 1.1);\n"
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "## GP regression in higher dimensions\n",
-    "Very little changes when the input space has more than one dimension. `X` is a NumPy array with one column for each dimension. The kernel can be set with `input_dim` equal to the number of columns of `X`. It is generally recommended that you set the `ARD` (Automatic Relevance Determination) parameter to `True` to enable you to tune a different lengthscale for each dimension."
+    "GP regression in higher dimension\n",
+    "--\n",
+    "\n",
+    "Very little changes when the input space has more than one dimension. It is generally recommended that you set the `ARD` (Automatic Relevance Determination) parameter to `True` to enable you to tune a different lengthscale for each dimension. "
```

Comment:
The ARD Parameter doesn't exist anymore - now need to pass in an array of the right length to the kernel's `lengthscale` argument